tests/unit/entityInfo.spec.ts

```diff
@@ -31,17 +31,13 @@ describe('Entity Info component', () => {
     expect(wrapper.vm.$el.querySelector('#nr-header').style.display).toBe('none')
   })
 
-  it('renders the Name Request header when the EntityType(BC) is present and not the initial header', () => {
+  it('renders the Name Request header when the EntityType(BC) is present and not the initial header', async () => {
     wrapper.vm.$store.state.stateModel.nameRequest.entityType = 'BC'
-    expect(wrapper.vm.$el.querySelector('#nr-header').textContent)
-      .toContain('Incorporate a BC Benefit Company')
-    expect(wrapper.vm.$el.querySelector('#no-nr-header').style.display).toBe('none')
-  })
 
-  it('renders the Name Request header when the EntityType(CP) is present and not the initial header', () => {
```

Comment:
Why remove this?

Reply:
Not sure. was this part of the unit test fix @ochiu did this morning?

Reply:
@lekshmimallika-aot 
I updated the nameRequestInfo.spec.ts and the App.spec.ts as they were causing the checks to fail. Entity Info tests were running fine so I didn't touch them.

Reply:
Then please restore that unit test. Thanks!

Reply:
Restored